cores/arduino/stm32/usb/cdc/usbd_cdc_if.c

```diff
@@ -284,7 +286,9 @@ void CDC_resume_receive(void)
       /* Set new buffer */
       USBD_CDC_SetRxBuffer(&hUSBD_Device_CDC, block);
       USBD_CDC_ReceivePacket(&hUSBD_Device_CDC);
+      return true;
     }
+    return false;
   }
 }

```

Comment:
@makarenya In fact,  it misses a default return when `receivePended ` is true

Reply:
Yes. It's my mistake. But it doesn't affect any branch of code now. I make new PR to solve this issue.